deepspeed/runtime/engine.py

```diff
@@ -2740,6 +2745,19 @@ def load_checkpoint(self,
             if not success:
                 self.optimizer._restore_from_bit16_weights()
 
+        if self.zero_has_nvme_offload():
+            from shutil import copytree, disk_usage
+            offload_dir = self.optimizer.optimizer_swapper.swap_folder
+            offload_ckpt_dir = os.path.join(load_dir, tag, "offloaded_tensors")
```

Comment:
```suggestion
            offload_ckpt_dir = os.path.join(load_dir, tag, OFFLOADED_TENSORS_DIR)
```
Replace "offloaded_tensors" with symbolic, such as above, which should be defined in deepspeed/checkpoints/constants.py

Reply:
@eisene, the PR looks great to me as is. My only concerns are the hardcoded constants such as this one. Do you have bandwidth to address at this time? Thanks!

Reply:
@tjruwase this seems very straightforward, I'll do it soon.

Reply:
@eisene, thanks. I will merge this now as it is not a reason to hold up this useful feature.